src/scripts/items/ItemHandler.ts

```diff
@@ -6,7 +6,7 @@ class ItemHandler {
     static amount: KnockoutObservable<number> = ko.observable(1);
     public static amountToUse = 1;
 
-    public static useItem(name: string, amount = 1): boolean {
+    public static useItem(name: string, amount = this.getAmount()): boolean {
```

Comment:
This probably shouldn't be changed, as you can just call `useItem` method with any amount you want to use from a different method instead.

Such as the battleItem uses this:
```js

                            event: {
                                click: function(){ItemHandler.useItem($data, EffectEngineRunner.amountToUse())}
                            },
```